packages/sdk/src/blockFinder.ts

```diff
@@ -29,9 +30,16 @@ export default class BlockFinder<T extends { number: number; timestamp: number |
     // Check the first block. If it's grater than our timestamp, we need to find an earlier block.
     if (this.blocks[0].timestamp > timestamp) {
       const initialBlock = this.blocks[0] as WithoutStringTimestamp<T>;
-      const cushion = 1.1;
-      // Ensure the increment block distance is _at least_ a single block to prevent an infinite loop.
-      const incrementDistance = Math.max(await estimateBlocksElapsed(initialBlock.timestamp - timestamp, cushion), 1);
+      // We use a 2x cushion to reduce the number of iterations in the following loop and increase the chance
+      // that the first block we find sets a floor for the target timestamp. The loop converges on the correct block
+      // slower than the following incremental search performed by `findBlock`, so we want to minimize the number of
+      // loop iterations in favor of searching more blocks over the `findBlock` search.
+      const cushion = 2;
```

Comment:
Cushion of 2 is effectively a 3x multiplier (the cushion is a percentage that is added to 1), but I think that's fine!

Reply:
good point will revert to 1